Wiktionary_scraper.py

```diff
@@ -58,26 +57,31 @@ def pronunciations_from_wiktionary_french(word):
     pronunciation = pronunciation.replace("[","")
     pronunciation = pronunciation.replace("]","")
     pronunciation = pronunciation.replace("/","")
-
     pronunciation = pronunciation.replace("\\","")
+    pronunciation = "/%s/" %pronunciation
 
     return [pronunciation]
 
+
 def pronunciations_from_wiktionary_italian(word):
     print("Fetching pronunciations for: %s\n" %word)
     soup = BeautifulSoup(requests.get('https://it.wiktionary.org/wiki/%s' % word).content, 'lxml')
 
     try:
         # In Italian there is only one pronunciation
         pronunciation = soup.find('span', class_="IPA").text
-        
-         # regex = re.compile('^[/\[\\].*[/\\\]]$')
-        pronunciation = "/"+pronunciation.replace("\\","")+"/"
-        return [pronunciation]
-
     except AttributeError:
         print("Can't find pronunciation for", word)
+        pronunciation = ""
+
+    # regex = re.compile('^[/\[\\].*[/\\\]]$')
+    pronunciation = pronunciation.replace("[","")
+    pronunciation = pronunciation.replace("]","")
+    pronunciation = pronunciation.replace("/","")
+    pronunciation = pronunciation.replace("\\","")
+    pronunciation = "/%s/" %pronunciation
```

Comment:
same here. plus this code is copy-pasted. put it in the text cleaning module as a function and defines it only once.